cloudbuild.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
```

Comment:
Is there any docs on this image? What it does/why we'd want to use it?

Reply:
This is my cloud build yaml that I find useful for testing- I am running paralell builds - on Travis and Cloud Build from our GCP project - Cloud Build is much faster as I am using 8 CPU High Memory machine and run all builds in parallel on it. We are using it also for integration testing with GCP operators and likely I want to make our integration tests work using the same image travis-ci uses, but this is something I will not include in the final PR.

Reply:
I will prefer to keep that one for now till we are closer to merge. It helps me to see if things are working fine also in cloud build. I will keep that conversation open.